plugins/VLC/__init__.py

```diff
@@ -812,14 +813,14 @@ def Configure(self, host="localhost", port=1234, showFeedbackEvents=True):
         'VolumeUp',
         'Volume Up',
         'Volume up',
-        'volup'
+        'volup 1'
     ),
     (
         ActionPrototype,
         'VolumeDown',
         'Volume Down',
         'Volume down',
-        'voldown'
+        'voldown 1'
     ),
```

Comment:
Please add a short comment about changed/fixed actions to the PR description. (And maybe replace `.)` with `-` which will be formated as list.)

Reply:
Thanks for the hint, I updated the initial comment accordingly.

Merry Christmas!